docs/Contributing.asciidoc

```diff
@@ -335,7 +335,57 @@ variable +OPENQA_BASEDIR+. The default value is +/var/lib+.
 
 == Running tests of openQA itself
 
-To execute the testsuite locally, use +make test+. It is also possible to run a
+There's two ways of executing the testsuite locally:
+
+1. with docker
+
+  The goal of running the tests with docker is to have consistent tests results (as sometimes the tests have different outcomes because 
+  missing packages or different package versions amongst other reasons). This is the preferred way if the user wants to run a full test
+  battery or if it needs to setup a test database
```

Comment:
Aren't you replacing the local `make test` run in travis with the docker way? This would mean we recommend to the new contributors a way to test locally in a way that is not tested anymore in a CI job? Also, for documentation purposes I would prefer if we can keep "simple" instructions first and mention a secondary way as "advanced".

Reply:
`make test` still works

The new contributors can still test both ways, but sometimes some errors happen in travis and not locally because of errors like, different packages and so on. These errors are hard to debug.

Using a docker image, we can guarantee that the contributor can expect the same outcome as the travis job., but it's not required to use docker. ;-)

Reply:
I would leave make test to run the testsuite locally, while have make docker-test for it and travis using it.

Reply:
We recommended before to make test on TW or Redhat but ran it on Travis. This changes introduces an additional way to do the same as travis does. And the experience in OBS team shows that developers migrate to use 'the travis way' as default